deploy/backplane/acs/01-acs-admins-cluster-aggregate.ClusterRole.yaml

```diff
@@ -40,3 +40,101 @@ rules:
   - get
   - list
   - watch
+# ACS SRE can view ingresses, ingress controllers, dnsrecords and publishing strategies
+- apiGroups:
+  - operator.openshift.io
+  resources:
+  - ingresscontrollers
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ingress.operator.openshift.io
+  resources:
+  - dnsrecords
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - cloudingress.managed.openshift.io
+  resources:
+  - publishingstrategies
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
Out of curiosity, what is this needed for? I tried looking at the linked Jiras but it wasn't clear why each resource is being requested

Reply:
Hey, I'll add a link to this in the PR description, it explains the change set: https://docs.google.com/document/d/1lyzFjK51py6o62zS5ErPFLbCNVJfq4e5PvVI2Y2z0Mg/edit

Reply:
Thanks. I took a look but I'm still not understanding why access to PublishStrategy is needed

Reply:
I can't find why, I'll remove the ask